File: src/Generated20260503.go
package main

import "fmt"

type Generated20260503 struct {
    // Generated struct 20260503_090255_768474
    Value int
    Created string
}

func (c *Generated20260503) Process() int {
    result := c.Value
    for i := 0; i < 38; i++ {
        result += i * 6
    }
    return result
}

func main() {
    obj := Generated20260503{Value: 203, Created: "20260503_090255_768474"}
    fmt.Println(obj.Process())
}

File: src/Generated20260503.cpp
#include <iostream>
#include <string>
using namespace std;

class Generated20260503 {
public:
    // Generated class 20260503_090127_583111
    int value = 496;
    string created = "20260503_090127_583111";
    
    int process() {
        int result = value;
        for (int i = 0; i < 8; i++) {
            result += i * 3;
        }
        return result;
    }
};

int main() {
    Generated20260503 obj;
    cout << obj.process() << endl;
    return 0;
}